content/articles/2017-08-tidyr-0.7.0.Rmd

```diff
@@ -0,0 +1,214 @@
+---
+title: tidyr 0.7.0
+slug: tidyr-0.7.0
+description: >
+  The next installment of tidyr is finally on CRAN! This version
+  brings tidy eval to a crucial component of the tidyverse workflow.
+author: Lionel Henry
+date: 2018-08-15
+photo:
+  url: https://unsplash.com/photos/8tem2WpFPhM
+  author: Radek Grzybowski
+categories: [packages]
+---
+
+```{r setup, include = FALSE}
+library("tidyr")
+library("dplyr")
+knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
+```
+
+We are happy to announce that tidyr 0.7.0 is now available on
+CRAN. This update makes tidyr compatible with the tidy evaluation
+features introduced in dplyr 0.7.0. It also uses the new tidyselect
+package as selection backend. The switch introduces some changes in
+the way selecting arguments are interpreted in order to make your
+tidyr code more robust.
+
+Install the latest version of tidyr with:
+
+```{r, eval = FALSE}
+install.packages("tidyr")
+```
+
+You can read about the complete set of changes
+at <https://github.com/tidyverse/tidyr/releases/tag/v0.7.0>.
+
+
+### New selection rules
+
+Following the switch to tidyselect, selecting functions are now
+stricter in their arguments An example of selecting function is
```

Comment:
An example -> . For example, take `gather()` and its `...` argument. Consider the following code: